assets/js/admin-order.js

```diff
@@ -364,7 +364,11 @@ jQuery(function ($) {
 
 				const updateView = (amount, percentage) => {
 					const discountedTotalAmount = totalAmount - amount;
-					newTotalAmountEl.val(`${discountedTotalAmount.toFixed(2)} SEK`);
+					// Get currency from the old price value.
+					const oldPrice = newTotalAmountEl.val();
+					const currencyMatch = oldPrice.match(/([A-Z]{3}|[^\d\s.,]+)/);
+					const currency = currencyMatch ? currencyMatch[0] : '';
+					newTotalAmountEl.val(`${discountedTotalAmount.toFixed(2)} ${currency}`);
```

Comment:
Suggestion: WooCommerce actually has logic for this that you can reuse on admin pages.

In Qliro we already use this for the frontend logic we have for the return fees. You can see the implementation of that [here](https://github.com/krokedil/qliro-for-woocommerce/blob/master/assets/js/admin-order.js#L276-L294) in the same file. You should be able to use that already, especially since we already pass the properties needed to the JavaScript file anyway when enqueueing it.